plugins/eddn.py

```diff
@@ -714,7 +714,7 @@ def filter_localised(d: Mapping[str, Any]) -> OrderedDictT[str, Any]:
                 this.commodities = this.outfitting = this.shipyard = None
                 this.marketId = entry['MarketID']
 
-            path = pathlib.Path(str(config.get('journaldir') or config.default_journal_dir)) / f'{entry["event"]}.json'
+            path = pathlib.Path(str(config.get_str('journaldir') or config.default_journal_dir)) / f'{entry["event"]}.json'
```

Comment:
again, pass default to get_str() rather than the 'or'.

Reply:
Updated